tests/Application/config/packages/sylius_invoicing.yaml

```diff
@@ -1,23 +1,23 @@
 imports:
     - { resource: "@SyliusInvoicingPlugin/Resources/config/config.yml" }
-            
-sylius_invoicing:
+
+sylius_resource:
     resources:
-        billing_data:
+        sylius_invoicing_plugin.billing_data:
```

Comment:
You should add a note to UPGRADE file about changing the way of overwriting resource classes